graph/R-GAT/tools/accuracy_igbh.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import numpy as np
+import torch
+import json
+import os
+
+
+def get_args():
+    """Parse commandline."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--mlperf-accuracy-file", required=True, help="path to mlperf_log_accuracy.json"
+    )
+    parser.add_argument(
+        "--dataset-path",
+        default="igbh",
+        help="Path to IHGB dataset",
+    )
+    parser.add_argument(
+        "--dataset-size",
+        default="full",
+        choices=["tiny", "small", "medium", "large", "full"]
+    )
+    parser.add_argument(
+        "--verbose",
+        action="store_true",
+        help="verbose messages")
+    parser.add_argument(
+        "--output-file", default="coco-results.json", help="path to output file"
+    )
+    parser.add_argument(
+        "--dtype",
+        default="int32",
+        choices=["float32", "int32", "int64"],
+        help="data type of the label",
+    )
+    args = parser.parse_args()
+    return args
+
+def load_labels(base_path, dataset_size, use_label_2K = True):
+    # load labels
+    paper_nodes_num = {
+        "tiny": 100000,
+        "small": 1000000,
+        "medium": 10000000,
+        "large": 100000000,
+        "full": 269346174,
+    }
+    label_file = (
+        "node_label_19.npy" if not use_label_2K else "node_label_2K.npy"
+    )
+    paper_lbl_path = os.path.join(base_path, "paper", label_file)
```

Comment:
Hi @pgmpablo157321 , would it be better to follow the same pattern as [here](https://github.com/mlcommons/inference/blob/d28a53009c4390cc43ccde00fba160b9fbf4c30e/graph/R-GAT/dgl_utilities/feature_fetching.py#L230) to maintain uniformity in the dataset path being passed to `main.py` and `accuracy_igbh.py`?

Reply:
Correct, just updated the path